content/applications/finance/sign.rst

```diff
@@ -55,66 +79,84 @@ similar acts instead.
-  field.
-- **Text**: users enter text on a single line.
-- **Multiline Text**: users enter text on multiple lines.
-- **Checkbox**: users can tick a box (e.g.,to mark their approval or consent).
-- **Selection**: users choose a single option from a variety of options.
+You can select an existing :guilabel:`Field` by clicking on it or you can :guilabel:`Create` a new
+one. First, edit the :guilabel:`Field Name`. Then, select your :guilabel:`Field Type`:
 
-Next, you have the option to auto-complete fields for users based on their Odoo profile information
-by using *Automatic Partner Field*. To this end, the *Name*, *Email*, *Phone*, and *Company* fields
-are preconfigured in your database.
+- :guilabel:`Signature`: users are prompted to enter their signature either by drawing it,
+  generating an automatic one based on their name, or uploading a local file (usually an image).
+  Each subsequent :guilabel:`Signature` field then reuses the data entered in the first field.
+- :guilabel:`Initial`: users are prompted to enter their initials, in a similar way to the
+  :guilabel:`Signature` field.
+- :guilabel:`Text`: users enter text on a single line.
+- :guilabel:`Multiline Text`: users enter text on multiple lines.
+- :guilabel:`Checkbox`: users can tick a box (e.g.,to mark their approval or consent).
+- :guilabel:`Selection`: users choose a single option from a variety of options.
 
-.. note::
-   Users can freely edit auto-completed fields.
+The :guilabel:`Auto-fill Partner Field` is the technical name of the field on the partner model to
+auto-complete this signature field at the time of signature.
 
-.. image:: sign/field-example.png
-   :align: center
-   :alt: Company field example in Odoo Sign
-
-You can then change the size of the field by editing the *Default Width* and *Default Height*. Both
-sizes are defined as a percentage of the full-page expressed as a decimal, with 1 equalling the
-full-page's width or height. By default, the width of new fields you create is set to 15% (0.150)
-of a full-page's width, while their height is set to 1.5% (0.015) of a full-page's height.
+.. note::
+   Auto-completed values are suggestions and can be modified as required by the person signing the
+   document.
 
-Next, write a *Tip*. Tips are displayed inside arrows on the left-hand side of the user's screen
-during the signing process. You can also use a *Placeholder* text to be displayed inside the field
-before it is completed.
+The size of the fields can also be changed, by editing the :guilabel:`Default Width` and
+:guilabel:`Default Height`. Both sizes are defined as a percentage of the full-page expressed as a
+decimal, with 1 equalling the full-page’s width or height. By default, the width of new fields you
+create is set to 15% (0.150) of a full-page’s width, while their height is set to 1.5% (0.015) of a
+full-page’s height. Next, write a :guilabel:`Tip`. Tips are displayed inside arrows on the left-hand
```

Comment:
Next, write a :guilabel:`Tip`.
What is this? are you introducing a new paragraph is so maybe go to the line before starting the sentence